sdcm/cluster.py

```diff
@@ -3510,7 +3510,8 @@ def is_table_has_data(self, session, table_name: str) -> (bool, Optional[Excepti
         Checking if a result page with any data is received.
         """
         try:
-            result = session.execute(SimpleStatement(f"SELECT * FROM {table_name}", fetch_size=10))
+            result = session.execute(SimpleStatement(f"SELECT * FROM {table_name} USING TIMEOUT 300s", fetch_size=10),
+                                     timeout=300)
             return bool(len(result.one())), None
```

Comment:
what's about fix this one ? and removing len ?

Reply:
@fruch , `len` is removed now.